lib/vanagon.rb

```diff
@@ -1,5 +1,8 @@
+require 'time'
+
 LIBDIR = File.expand_path(File.dirname(__FILE__))
 VANAGON_ROOT = File.join(File.expand_path(File.dirname(__FILE__)), "..")
+BUILD_TIME = Time.now
```

Comment:
Quick note: maybe cast this to a string up front, so you don't have to worry about converting it down the line. You don't do anything directly with the object, so don't hang onto a Time object longer than you really have to.